lib/elixir/lib/module.ex

```diff
@@ -861,6 +861,15 @@ defmodule Module do
       end
 
   """
+  def put_attribute(module, key, value)
+
+  def put_attribute(module, key, value) when key in [:moduledoc, :typedoc, :doc] do
+    unless match?({line, _} when is_integer(line), value) do
+      raise ArgumentError, "expected #{inspect(key)} attribute given in the {line, doc} format, got: #{inspect(value)}"
```

Comment:
Still requiring to put a tuple back. But making the errormessage a little bit better...